src/doc/ko/a_tour_of_sage/index.rst

```diff
@@ -0,0 +1,135 @@
+===========================
+Sage에 오신 것을 환영합니다
+===========================
+
+이것은 Sage와 그것을 계산기로 사용하는 것에 대한 간단한 안내입니다.
+
+Sage의 명령 줄은 ``sage:`` 프롬프트 메시지로 시작합니다. 다음 예제들을 실험하려면
+프롬프트 메시지 다음 부분만 입력하면 됩니다.
+
+::
+
+    sage: 3 + 5
+    8
+
+Jupyter 노트북에서 Sage를 사용하는 경우, 마찬가지로, 프롬프트 메시지 이후의
+모든 내용을 입력 셀에 넣고 :kbd:`Shift-Enter` 를 눌러 해당 출력을 얻으세요.
+
+``^`` 기호는 거듭제곱을 의미합니다.
+
+::
+
+    sage: 57.1^100
+    4.60904368661396e175
+
+우리는 Sage에서 :math:`2 \times 2` 행렬의 역행렬을 계산합니다.
+
+::
+
+    sage: matrix([[1, 2], [3, 4]])^(-1)
+    [  -2    1]
+    [ 3/2 -1/2]
+
+여기에서 간단한 함수의 적분을 계산합니다.
+
+::
+
+    sage: x = var('x')   # δημιουργίας συμβολικής μεταβλητής
```

Comment:
These need changing too :) (I know this is a draft PR, just saying)

Reply:
Yes, it is a draft yet. 

I made these Korean versions using ChatGPT from your Greek versions. I will now compare the Korean versions with English versions and make them readable. Along the way, I could "review" your Greek versions :-)